SwiftLint/swiftlint.yml

```diff
@@ -253,6 +253,16 @@ custom_rules:
     regex: '(?-smxi)(func get[A-Z0-9][\w]+?(\())'
     message: "You should not prefix your functions with the word get."
     severity: warning
+  body_indentation:
```

Comment:
The name is quite generic for the the specific rule. Something like `multiline_if_statement_indentation_violation` would avoid conflicts to other rules which we or SwiftLint might add regarding body indentation